patches/server/1076-Fix-Player.setPlayerListOrder-to-send-update-to-clie.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: 96DarkCode96 <96darkcode96@gmail.com>
+Date: Tue, 10 Dec 2024 15:45:47 +0100
+Subject: [PATCH] Fix Player.setPlayerListOrder to send update to clients
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 20f64850cb42d986358f01ffcdb42187e1684bca..b60f547c8616e9d4a858e79d036712f311152aea 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -606,6 +606,12 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+         Preconditions.checkArgument(order >= 0, "order cannot be negative");
+ 
+         this.getHandle().listOrder = order;
++        if (getHandle().connection == null) return; // Updates are possible before the player has fully joined
++        for (ServerPlayer player : server.getHandle().players) {
++            if (player.getBukkitEntity().canSee(this)) {
```

Comment:
This also needs to consider the isListed API

Reply:
should be updated

Reply:
Does it? Being unlisted doesn't mean there's no player data. When re-listed, it only updates the listed status, nothing else

Reply:
Yea, no, I totally brainfarted, I misread some stuff, I was wrong here

Reply:
Personally I haven't used isListed API ever in my codes, so really don't know what it does in background, but as I went through, this packet is always send regardless of that isListed API.

Reply:
yea, I was wrong to say that this should consider the isListed API, that last commit needs reverting

Reply:
i'll try to revert that, i'm really fighting with PR system (first PR that I've ever done). But I need to learn that.

Reply:
You should just be able to `git revert 9c855b2` and then push